fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/debug/DebuggerWithSymbolTables.java

```diff
@@ -79,7 +79,11 @@ public PlanContext getPlanContext() {
 
     @Override
     public boolean isSane() {
-        return true;
+        //
+        // Report insanity here which then causes all sanity checks to be run which may be CPU-intensive. Deactivate
+        // this behavior by returning true if performance is measured.
```

Comment:
Should this have some kind of environmental control then?

Reply:
I wanted to have this controlled vi a java property. Not done yet.